website/src/data/components/bar/meta.yml

```diff
@@ -1,76 +1,78 @@
 flavors:
-  - flavor: svg
-    path: /bar/
-  - flavor: canvas
-    path: /bar/canvas/
-  - flavor: api
-    path: /bar/api/
+    - flavor: svg
```

Comment:
If possible, I'd prefer to keep the formatting as is, as it's not nice for git history.

Reply:
I'm sorry it was prettier